### DISCUSSION TITLE
How to use multiline tooltips?

### DISCUSSION THREAD

**Discussion by sekzerd** [quasarframework/quasar#17574] [Ideas / Proposals] [answered]
Hi, I am using quasar. It's seem like Tooltip not support multiline text, and i can't find any option in docs. If It's possible add multiline text in Tooltip?

**Comment by sekzerd** [comment, marked as answer]
Founded solution
like this

```html
<div v-for="item in items">
    <q-tooltip anchor="top middle" self="center middle">
        <div v-for="lines in get_tooltip(item)">
            {{ lines }}
        </div>
    </q-tooltip>
</div>
```

```js
let items = [{ label : "foo", value : 0 },{ label : "bar", value : 1 }];
function get_tooltip(item){
    let ret = [];
    ret.push("label:"+item.label);
    ret.push("value:"+item.value);
    return ret;
}

**Comment by lupodellasleppa** [comment]
for anyone who stumbles here while searching around the internet:

Just use `white-space: pre` (or `pre-wrap`, or `pre-line`, depending on your needs) on the tooltip element, and insert an html line break element (`<br >`) to, well... break the line

```vue
<q-tooltip style="white-space: pre">
  tool <br />
  tip
</q-tooltip>
```

ref: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space
